doc/modules/gaussian_process.rst

```diff
@@ -487,7 +487,7 @@ number of dimensions as the inputs :math:`x` (anisotropic variant of the kernel)
 The kernel is given by:
 
 .. math::
-   k(x_i, x_j) = \text{exp}\left(-\frac{1}{2} d(x_i / l, x_j / l)^2\right)
+   k(x_i, x_j) = \text{exp}\left(- \frac{d(x_i, x_j)^2}{2l^2} \right)
```

Comment:
For the rational quadratic, the doc already uses `1/l d(x_i, x_j)`, however, there, it's also like this in the code.
I originally didn't touch the Matern formula because it looked intimidating :sweat_smile: but since I now already simplified it removing the small \gammas, I can also adapt it to use `d(x_i, x_j)`. But not sure if you prefer to keep it close to the code or to use everywhere the same notation.

Reply:
I'm happy to use a similar notation in different places.